source/conf.py

```diff
@@ -0,0 +1,72 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'PARL'
+copyright = '2019, nlp-ol@baidu.com'
+author = 'nlp-ol@baidu.com'
+
+# The full version, including alpha/beta/rc tags
+release = '1.1'
```

Comment:
Should use global version information, like parl.\_\_version\_\_ or reference: https://github.com/PaddlePaddle/PARL/blob/255ef4f7877195d9f57a3cb728323562457b34b6/setup.py#L59